training/src/main/scala/com/airbnb/aerosolve/training/TrainingUtils.scala

```diff
@@ -179,6 +203,16 @@ object TrainingUtils {
     }
   }
 
+  def getLabel(example: Example, loss: String, rankKey: String, threshold: Double): Int = {
+    val fv = example.getExample.get(0)
+    val label: Int = if (loss == "regression") {
+      TrainingUtils.getLabel(fv, rankKey).toInt
```

Comment:
Since we know this is a classification label, and I use `int` to create the maps (classweights and downsample), that will make the context clearer and can usually prevent potential bugs.